Compiler warning/error for "va_start"

There is a compiler warning/error that occurs on build.  It is:
```
...\Arduino_DebugUtils.cpp: In member function 'void Arduino_DebugUtils::print(int, const __FlashStringHelper*, ...)':
...\Arduino_DebugUtils.cpp:86:3: warning: second parameter of 'va_start' not last named argument [-Wvarargs]
va_start(args, fmt_str.c_str());
```

From my investigation, is seems that line 86 (of "Arduino_DebugUtils.cpp") is:
```
va_start(args, fmt_str.c_str());
```
and should instead be:
```
va_start(args, fmt);
```

I'm not overly familiar with that macro, but this version does stop the error and seems to function correctly when I run the examples.

--- a/src/Arduino_DebugUtils.cpp
+++ b/src/Arduino_DebugUtils.cpp
@@ -83,7 +83,7 @@ void Arduino_DebugUtils::print(int const debug_level, const __FlashStringHelper 
   String fmt_str(fmt);
 
   va_list args;
-  va_start(args, fmt_str.c_str());
+  va_start(args, fmt);
   vPrint(fmt_str.c_str(), args);
   va_end(args);
 }

